blazor/accumulation-chart/title-and-sub-title.md

```diff
@@ -93,6 +93,48 @@ The [Title](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Charts.SfAcc
 ![Customizing Title in Blazor Accumulation Chart](images/title/blazor-accumulation-chart-title-customization.png)
 {% previewsample "https://blazorplayground.syncfusion.com/embed/rtBUMVCpMmDIcLry?appbar=false&editor=false&result=true&errorlist=false&theme=bootstrap5" %}
 
+### Title Position
```

Comment:
This feature is available for Chart and accumulation chart. But the topic is added only for accumulation chart.